src/target/adiv5_swd.c

```diff
@@ -75,11 +75,18 @@ static void dormant_to_swd_sequence(void)
 	 * §5.3.4 Switching out of Dormant state
 	 */
 
-	DEBUG_INFO("Switching out of dormant state into SWD\n");
-
 	/* Send at least 8 SWCLKTCK cycles with SWDIOTMS HIGH */
 	swd_line_reset_sequence(false);
+
+	/* If the target is both Jtag and SWD with Jtag as default, switch Jtag->DS first. 
```

Comment:
Please put the opening `/*` on its own line like in the comment blocks above this.

Reply:
done